app/views/articles/show.html.erb

```diff
@@ -155,11 +155,12 @@
                 <% if @article.published_timestamp.present? %>
                   <%= local_date(@article.published_timestamp, show_year: @article.displayable_published_at.year != Time.current.year) %>
                 <% end %>
-                <% if @second_user.present? %>
-                  <em>with <b><a href="<%= @second_user.path %>"><%= @second_user.name %></a></b></em>
-                <% end %>
-                <% if @third_user.present? %>
-                  <em> and <b><a href="<%= @third_user.path %>"><%= @third_user.name %></a></b></em>
+                <% if @article.co_author_ids.present? %>
+                  <% if @article.co_author_ids.count == 1 %>
+                    <em>with <b><a href="<%= @article.co_authors.first.path %>"><%= @article.co_authors.first.name %></a></b></em>
+                  <% else %>
+                    <em>with <%= @article.co_author_name_and_path.html_safe %></em>
```

Comment:
why make a distinction here?

Reply:
Great question! I thought this distinction was necessary for formatting reasons and was needed to create a coherent sentence, but now I'm realizing that the way I set the logic up in the decorator makes so that I don't even need the distinction anymore, so I've since removed this conditional. 😄